### PR TITLE
feat(pf-icons): add all PF icons and update font awesome pckg

### PR DESCRIPTION
**Pull request by karelhala**
affects: @patternfly/react-icons

resolves: https://github.com/patternfly/patternfly-next/issues/730 

https://5ba9f1b873f2cf3535ebccc7--zen-swanson-2d3350.netlify.com/styles/icons

Icon definitions were taken from https://github.com/patternfly/patternfly/blob/master/src/fonts/PatternFlyIcons-webfont.svg?short_path=2c3738f

### CODE REVIEW

**Comment by karelhala** [comment]
@LHinson I have a question, since this resolves issue in PF-next do you think I should move the icon definition to PF-next? Since that would make more sense and others could import the icon defintion.

**Comment by dmiller9911** [comment]
Not sure about Leslie's and the teams thoughts, but the svgs or the path data should live in core and then pulled in. Unless the team has went all in on react they need to be offered by core so other frameworks or HTML/css consumers can use them.

**Comment by karelhala** [comment]
@dmiller9911 yeah the paths should be part of core, it was convenient to copy them from SVG font definition over here into JS file. But I guess they can be moved to PF-next into JSON file and imported here.

**Comment by LHinson** [comment]
@karelhala there is an issue over on pf-next here: https://github.com/patternfly/patternfly-next/issues/730

Agreed that we should add to core first. Would you be interested/willing to pick up this issue first?

File: packages/react-icons/build/pfIcons.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  users,
+  info,
+  filter,
+  key,
+  ...icons
+} = require('@patternfly/patternfly-next/icons/pf-icons');
+
+module.exports = {
+  pfIcons: {
+    'alt-save': save,
```
Comment (dmiller9911) [inline packages/react-icons/build/pfIcons.js]: these might be better as `save-alt` it un-groups them, but it does keep them next to the related icon when it comes to intellisense and the examples page.

Comment (karelhala) [reply packages/react-icons/build/pfIcons.js]: Yeah, good point!

**Comment by karelhala** [comment]
@jschuler @amarie401 looks like travis thought that the build is incorrect so I had to update the PR by calling `git commit --amend` and now GH thinks that your reviews are invalid. Can you re-review once more please?

**Pull request merged**